### ISSUE TITLE
About VGG16 pretrained model

### ISSUE THREAD

**Issue by zijundeng** [wkentaro/pytorch-fcn#52]
Hi @wkentaro , I see that you use the vgg16 pretrained model from caffe. I wonder that why you do not use a pretrained [one](https://github.com/pytorch/vision/blob/master/torchvision/models/vgg.py) provided by torchvision. Is the pretrained one from caffe better?

Thanks.

**Comment by wkentaro** [comment]
I wanted to reproduce the same result as the original paper, and since pytorch uses different implementation for vgg (pytorch normalizes pixel value from 0 to 1), I decided to copy weights from caffe.
I have not trained fcn using vgg model in torchvision.
Does the model achieve the same result as the original vgg?

**Comment by zijundeng** [comment]
I have tried to use the pretrained vgg from torchvision for several times but failed to achieve the same result.

**Comment by zijundeng** [comment]
The mean iu with the pretrained vgg from torchvision is 60~62. Maybe I haven't found a good set of hyperparameters.

**Comment by wkentaro** [comment]
It seems that this issue is not the problem of pytorch-fcn in actual.
I think the torchvision vgg does not achieve the same result on imagenet, either. If I am wrong, please report.

Please request me to reopen this if you need.

**Comment by dillondavis** [comment]
@ZijunDeng Did you ever find good hyperparameters for the torchvision vgg16? I'm having some issues training it

**Comment by hylovezhang** [comment]
Can you help me ? why i use the vgg16_from caffe model to train , the result of testing is zero? @zijundeng
